### ISSUE TITLE
Invalid field-name return with locales

### ISSUE THREAD

**Issue by ghost** [Mgiepz/saiku-adhoc#155]
Hello Mgiepz and others,

On "Saiku-Adhoc", in the case where the "locale" default is replaced by another, the fields displayed in areas Column / Groups / Filters take name as the id field. And as for the headers of columns in the reports.

For security reasons, I am unable to provide the xmi file generated by Pentaho Metadata Editor: S But I'm still at your disposal to answer your questions as far as possible.

Thank you very much.

**Comment by Mgiepz** [comment]
I am not sure if I understand your problem. If you mean that the report doesnt change the column headers and the fields in the dropzones when you switch the locale after you created the report, then this is the expected behaviour. Currently the Column's identifier is the same as its caption. Otherwise the CDF-Tableview would show the internal id which does not mean anything to the user.

**Comment by Pierre-Antoine** [comment]
Hi,

I had the same problem: I set the labels of my fields with the locale fr_FR and all I got on the report was the IDs... When I filled the en_US locale, then it worked correctly. I am wondering if Saiku Reporting is able to handle several locales though, it might be a serious limitation...

**Comment by Mgiepz** [comment]
Yes it is able to handle locales. I have several examples where it is 
working.

Am 19.12.2012 11:12, schrieb Pierre-Antoine:

**Comment by Pierre-Antoine** [comment]
Good to know! What is your configuration to make it work then?

**Comment by Pierre-Antoine** [comment]
This issue is still unsolved (or undocumented)... Any tip?
